backend/monitor/src/main.rs

```diff
@@ -0,0 +1,65 @@
+extern crate winapi;
```

Comment:
Rust version not planned corrently.
Tis also jsut seems to be a functionality copy in rust without any use of its memory safe features.
If you are considering a rewrite, use a different architecture than the current python engine